com.github.simonwhitaker.Gibo.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.github.simonwhitaker.Gibo
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
```

Comment:
Update runtime to 19.08

Reply:
This is not specifically required, but would be good to check.

Reply:
Done, works as well